src/types/FilterTodosBy/FilterTodosBy.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterTodosBy {
```

Comment:
The base filename and primary named export do not match. Consider renaming the file to 'FilterTodosBy.ts' or updating the primary named export to match the filename.